src/main/java/uk/gov/hmcts/cmc/claimstore/rpa/MoreTimeRequestedNotificationService.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.cmc.claimstore.rpa;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.events.response.MoreTimeRequestedEvent;
+import uk.gov.hmcts.cmc.claimstore.rpa.config.EmailProperties;
+import uk.gov.hmcts.cmc.claimstore.utils.DocumentNameUtils;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.email.EmailAttachment;
+import uk.gov.hmcts.cmc.email.EmailData;
+import uk.gov.hmcts.cmc.email.EmailService;
+import uk.gov.hmcts.cmc.rpa.mapper.MoreTimeRequestedJsonMapper;
+
+import java.util.Collections;
+
+import static java.util.Objects.requireNonNull;
+
+@Service("rpa/more-time-requested-notification-service")
+@ConditionalOnProperty(prefix = "feature_toggles", name = "emailToStaff")
+public class MoreTimeRequestedNotificationService {
+
+    public static final String JSON_EXTENSION = ".json";
```

Comment:
Please make private.

Reply:
These class/classes are parked into different branch until requirements are more clear in conversation with PO